### DISCUSSION TITLE
Does xDrip work with Dexcom G6 One?

### DISCUSSION THREAD

**Discussion by ailuirazal** [NightscoutFoundation/xDrip#2534] [Q&A] [answered]
Hello guys, 
I have been looking for issues between Dexcom G6 **ONE** and I haven't found anything. If people already asked and it's a duplicate question please delete this!

I've recently seen that Dexcom has a site for Romania where they are selling some products called Dexcom G6 **ONE** sensors and transmitters: https://www.dexcom.com/ro-RO/shop/dexcom-one/

They look identical to the good ol' Dexcom G6 so I was wandering if you know if we can use these sensors with xDrip?

It would be great if we could buy the supplies directly from Dexcom! We would have some support and warranty 👍  something we haven't seen in this part of the world :) . 
Anyway, I'd appreciate some feedback about xDrip and Dexcom G6 One if any...if I'm a trailblazer I shall give it a try and come back with a feedback myself. 

My setup is as follows:
xDrip version: xDrip-plus-20221127-7352414.apk
Android version: 12
phone: OnePlus Nord from Oppo

Thanks!

**Comment by Navid200** [comment, marked as answer]
You can open a discussion and ask a question.  There is no problem if someone asked the same question before. 
You can open a discussions and rant if you need to.  As long as it is not offensive, go for it.
The reason we asked for discussions to be opened was that people did all of the above in issues.  Doing that with discussions is all good.  Again as long as there is no offense, bad language, spam, bullying, ....

xDrip has supported Dexcom ONE for quite a while: https://navid200.github.io/xDrip/docs/Dexcom-One

**Reply by ailuirazal** [reply]
Hey Navid200, 

Thanks for the link and info about the rant. Silly me, screaming in the forest 😮‍💨 GitHub was the place for the rants🎃 

That's a very good news that we can use xDrip and Dexcom G6 One 🥇 .  

I shall be testing it myself after I finish the sensors and transmitters I've bought from Czech Republic.

All the best!
Iulia

**Reply by thc0d33r** [reply]
@ailuirazal I'm using Dexcom G6 transmitter with Dexcome ONE sensors bought online from Romania. From what I see not all Dexcom ONE transmitters are supported by xDrip so a setup with Dexcom G6 transmitter and Dexcom ONE sensors seems ideal. Can you please tell me where did you purchase the transmitters from? As mine is about to expire and can't seem to find them anymore.

**Reply by ailuirazal** [reply]
Hello @thc0d33r ,
I'm currently using Dexcom G6 transmitter sensors bought online from  the Czech Republic. 

https://eshop.aimport.cz/kat/en/kontinualni-monitorace?country=CZ&currency=EUR

Unless you are located in the Czech Republic or Slovakia you cannot order from the site, but make an order via mail (write to  Ing. Martin Kuchta <mkuchta@aimport.sk> ) and they will send a payment link in USD and then deliver the products to your address with the DPP company.

The prices are as follows:
Prices in USD with VAT:

- Dexcom G6 transmitter – 75 USD
- box of 3 Dexcom G6 sensors – 225 USD
- shipping costs – 17 USD (free shipping for orders of 600 USD)

I'm planning to switch to Dexcom G6 ONE (both sensors and transmitters) in 3 months time aprox March 2023 (as I still have 3 sensors and a transmitter bought from aimport.cz from last year).

Hope it helps! 
Iulia

**Reply by thc0d33r** [reply]
@ailuirazal thank you very much for your comprehensive answer.

**Comment by ailuirazal** [comment]
Hello, 

I've just wanted to come back with a feedback. I've updated to the latest xDrip version and used a **Dexcom G6 One sensor and transmitter** for the past month. The only hiccup was the transmitter wasn't found for about 15 minute **after** the 2 hour warm up period, but after that it was business as usual. I'm currently at the 2nd Dexcom G6 One sensor 1st transmitter.
So @Navid200  thank you very much for your help! 👍
